apps/chat/src/components/Files/SelectFolderModal.tsx

```diff
@@ -129,7 +137,7 @@ export const SelectFolderModal = ({
       </SelectFolderHeader>
       <SelectFolderFooter
         handleNewFolder={handleNewFolder}
-        onSelectFolderClick={() => onClose(selectedFolderId)}
+        onSelectFolderClick={() => handleClose(selectedFolderId)}
```

Comment:
Are you sure that we need to clear error message here?

Reply:
Yes, I believe so. Selecting a folder also closes the window and should clear the error message, just as simply closing the window does. In both of these scenarios, we don't want the same error message to appear when we open the window the next time.